apiserver/facades/agent/instancemutater/lxdprofilewatcher_test.go

```diff
@@ -0,0 +1,476 @@
+// Copyright 2019 Canonical Ltd.
```

Comment:
moved from `core/cache/lxdprofilewatcher_test.go`
core tests are the same but ported to use the new go mocks